lib/common/options.c

```diff
@@ -585,6 +585,7 @@ add_desc(GString *s, const char *tag, const char *desc, const char *values)
                 locale = strtok(setlocale(LC_ALL, NULL), "_");
             }
 
+            g_string_append(s, "      ");
```

Comment:
Instead of always doing a fixed amount, let's pass a prefix argument to add_desc() with the spaces string to use (for both the English and NLS output), since some of the callers use fewer spaces.